python/lsst/ts/ATWhiteLightSource/chillerModel.py

```diff
@@ -210,22 +213,29 @@ def __str__(self):
         """
         print(type(self.component))
         await self.component.connect()
+        self.disconnected = False
         self.queue_task = asyncio.ensure_future(self.queueloop())
```

Comment:
I think it would be good to define `queue_task` and `watchdog_task` on `__init__` alongside `self.watchdogLoopBool` and `self.queueLoopBool`. They should probably initialize as `None`.

Then `self.watchdogLoopBool` and `self.queueLoopBool` should start as `False` and you can set them to `True` before scheduling the tasks. Furthermore, in `disconnect` set them back to `None` once you are done with them.